webapp/src/main/java/com/cmds/webapp/controllers/APIController.java

```diff
@@ -201,6 +235,7 @@ public Delivery getDeliveryStatus(@PathVariable("id") String id) throws IOExcept
         return deliveryRepo.findById(Long.valueOf(id)).orElse(null);
```

Comment:
This should return 200 not the delivery object. I just want to make sure the post request went through properly.

Reply:
Wrong method :)